src/components/sales/index.js

```diff
@@ -1,11 +1,210 @@
-import React from 'react'
+// @flow
 
-import SalesReport from '../sales-report'
-import Orders from '../orders'
+import type { OrdersOrderedType, OrderOrderedType } from 'constants/firebase'
 
-export default () => (
-  <div style={{ overflow: 'scroll' }}>
-    <SalesReport />
-    <Orders />
-  </div>
-)
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import _ from 'lodash'
+import * as React from 'react'
+import ReactList from 'react-list'
+
+import Flex from 'components/common/flex'
+import Button from 'components/common/button'
+import Spinner from 'components/common/spinner'
+import orderUtil from 'utils/order'
+import ORDER from 'constants/order'
+
+import OrderRow from './container/order-row'
+import style from './index.style.js'
+
+export type PropsType = {|
+  ordersOrdered: ?OrdersOrderedType,
+|}
+
+class Sales extends React.PureComponent<PropsType> {
+  render() {
+    const { ordersOrdered } = this.props
+
+    if (_.isEmpty(ordersOrdered)) {
```

Comment:
What would happen if you haven't sold anything yet? Would we have always showing spinner?

Reply:
Good catch!